aws-crt/native/src/event_loop_group.c

```diff
@@ -6,34 +6,17 @@
 
 #include <aws/io/event_loop.h>
 
-struct aws_crt_event_loop_group {
-    struct aws_event_loop_group native;
-};
-
 struct aws_crt_event_loop_group *aws_crt_event_loop_group_new(uint16_t max_threads) {
-    struct aws_crt_event_loop_group *crt_elg =
-        aws_mem_calloc(aws_crt_allocator(), 1, sizeof(struct aws_crt_event_loop_group));
-    if (!crt_elg) {
-        goto error;
-    }
-
-    if (aws_event_loop_group_default_init(&crt_elg->native, aws_crt_allocator(), max_threads)) {
-        goto error;
-    }
+    struct aws_event_loop_group *elg =
+        aws_event_loop_group_new_default(aws_crt_allocator(), max_threads, NULL /*shutdown_options*/);
 
-    /* Success */
+    /* Don't need CRT-specific struct, so just cast */
+    struct aws_crt_event_loop_group *crt_elg = (struct aws_crt_event_loop_group *)elg;
```

Comment:
Not sure whether or not we should **always** pass a unique "crt" type for the DLL API.

In other CRTs, we've often needed to wrap the original struct in another struct so we can add extra fields.
But for this particular class, we don't currently need extra fields. So do I keep the special "crt" type?

Feel free to vote "yeah totally we should always declare a unique type in the DLL API" or "this casting is dumb and confusing, wait until you actually need a unique type"

Reply:
I'd lean towards not having a special "crt" type when not needed.  Since these will not be part of an API that we need to keep stable (ie, these will always be api private), we can safely do that later if/when needed.  Not a strong feeling though so feel free to override my vote :-)